lib/templ.ml

```diff
@@ -897,13 +897,18 @@ let print_copyright conf =
       Output.print_sstring conf "<br>\n")
 
 let include_hed_trl conf name =
-  if conf.debug && name = "trl" then
-    begin
-      let query_time = (Unix.gettimeofday ()) -. conf.query_start in
-      Output.print_sstring conf
-        (Printf.sprintf "Query treated in <span id=\"q_time_c\">%.3f</span> seconds " query_time) ;
-      Output.flush conf ;
-    end ;
+  if conf.debug && name = "trl" then (
+    let query_time = Unix.gettimeofday () -. conf.query_start in
+    Output.print_sstring conf
+      (Printf.sprintf
+         {|Query treated in <span id="q_time_c">%.3f</span> seconds|} query_time);
+    Output.print_sstring conf "<script>\n";
+    Output.print_sstring conf
+      {|var q_time = document.getElementById("q_time_c").innerHTML;|};
+    Output.print_sstring conf
+      {|document.getElementById("q_time_d").innerHTML = q_time;|};
+    Output.print_sstring conf "</script>\n";
```

Comment:
Im not a fan of having jquery script in .ml files
do we really need it for debug?

also it is duplicated, maybe make a time_debug_script variable so you don't have to duplicate it and write it all in one string:
```ocaml
let time_debug query_time =
Printf.sprintf {|
Query treated in <span id="q_time_c">%.3f</span> seconds
<script>
var q_time = document.getElementById("q_time_c").innerHTML;
document.getElementById("q_time_d").innerHTML = q_time;
</script>
 |} 
query_time

Reply:
It is not jquery, but plain javaScript!
But yes, I will factorise.
With the cous_paths features, knowing the response time becomes significant, but yes, we could do away within later.

Reply:
ok